lib/barbeque/config.rb

```diff
@@ -0,0 +1,49 @@
+require 'erb'
+require 'yaml'
+
+module Barbeque
+  class Config
+    attr_accessor :exception_handler, :runner, :runner_options
+
+    def initialize(options = {})
+      options.each do |key, value|
+        if respond_to?("#{key}=")
+          public_send("#{key}=", value)
+        else
+          raise KeyError.new("Unexpected option '#{key}' was specified.")
+        end
+      end
+      init_runner_options
+    end
+
+    private
+
+    def init_runner_options
+      self.runner_options ||= {}
+      self.runner_options = Hash.new.tap do |symbolized|
```

Comment:
I don't want to do `symbolize_keys` here. When `runner_options` is `{ "hako_env" => { "FOO" => "BAR" } }`, it will be `{ hako_env: { FOO: "BAR" } }`. And `{ FOO: "BAR" }` will be passed to `Open3.capture3` and it will raise `TypeError`.